# empty request
---

# reply: the next room
string room
